plaidml/bridge/openvino/src/plaidml_plugin/ops/split.cpp

```diff
@@ -0,0 +1,51 @@
+// Copyright (C) 2020 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "plaidml_ops.hpp"
+
+#include "ngraph/opsets/opset.hpp"
+#include "ngraph/opsets/opset1.hpp"
+
+#include "plaidml/op/op.h"
+
+using namespace plaidml;          // NOLINT[build/namespaces]
+using namespace InferenceEngine;  // NOLINT[build/namespaces]
+
+namespace PlaidMLPlugin {
+
+static OpRegistration reg("split", [](const Context& ctx) {
+  auto* layer = dynamic_cast<ngraph::opset1::Split*>(ctx.layer);
+  IE_ASSERT(ctx.operands.size() == 2);
+  auto I = ctx.operands.at(0);
+  // operands.at(1) is unused, just read the Constant instead
+  auto splits = layer->get_num_splits();
+  ngraph::AxisSet axes;
+  auto axis_ngraph_op = std::dynamic_pointer_cast<ngraph::op::Constant>(layer->input_value(1).get_node_shared_ptr());
+  if (axis_ngraph_op) {
+    axes = axis_ngraph_op->get_axis_set_val();
+  } else {
+    THROW_IE_EXCEPTION << "Dynamic axis not currently supported by PlaidML plugin";
+  }
+  IE_ASSERT(axes.size() == 1);
+  auto axis = axes.to_vector()[0];
```

Comment:
Might be good to throw this into a utility since this appears to be something you'll need to do fairly often.